UI/main.html

```diff
@@ -21,6 +21,21 @@
         </a>
       </div>
       <div id="version_info">[% text("LedgerSMB [_1]", version) %]</div>
+      <script>
```

Comment:
It would be nice to understand why we need this. I think we managed to move all JS out of the various HTML pages to dojo widgets; this moves us away from that again. Can it be done in a/the dojo widget?

Reply:
LGTM is fixed. This should indeed be moved out of the page but, at least for a while, there should be a way to identify on which browser was the application running.

Reply:
Maybe we should show it with a special URL query string argument?